src/sentry/models/organizationcontributors.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import BoundedIntegerField, FlexibleForeignKey, Model, region_silo_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_model
+class OrganizationContributors(Model):
+    """
+    Tracks external contributors and their activity for an organization.
+    This model stores information about contributors associated with an
+    integration for a specific organization, including their external identity
+    and how many actions they have taken.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    organization = FlexibleForeignKey("sentry.Organization", on_delete=models.CASCADE)
+
+    integration_id = HybridCloudForeignKey("sentry.Integration", on_delete="CASCADE")
+
+    external_identifier = models.CharField(max_length=255, db_index=True)
+    alias = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
Does it make sense to have an FK to this table instead of having these fields? https://github.com/getsentry/sentry/blob/153511a3db57b6dc0259dbda70bb269f8d5d9076/src/sentry/models/commitauthor.py#L27-L40

Or is the format there totally different?